components/frontend/src/metric/MetricDetails.js

```diff
@@ -81,20 +81,23 @@ Buttons.propTypes = {
     url: string,
 }
 
-function fetchMeasurements(reportDate, metric_uuid, setMeasurements) {
+function fetchMeasurements(reportDate, metric_uuid, setMeasurements, setMeasurementsStatus) {
     get_metric_measurements(metric_uuid, reportDate)
         .then(function (json) {
-            if (json.ok !== false) {
-                setMeasurements(json.measurements)
-            }
```

Comment:
This is dead code: the API-server GET measurements endpoint does not have an ok flag.